draft-ietf-jose-hpke-encrypt.md

```diff
@@ -261,20 +261,24 @@ After verification:
 # Key Encryption
 
 HPKE based recipients can be added alongside existing `ECDH-ES+A128KW` or `RSA-OAEP-384` recipients because HPKE is only used to encrypt the content encryption key, and because the protected header used in content encryption is passed to HPKE as Additional Authenticated Data.
+This protected header is formatted as BASE64URL(UTF8(JWE Protected Header)).
 
 In HPKE JWE Key Encryption:
 
-- The protected header MUST NOT contain an "alg".
-- The protected header MUST contain an "enc" that is registered in both the IANA HPKE AEAD Identifiers Registry, and the IANA JSON Web Signature and Encryption Algorithms Registry.
+- The JWE protected header MUST NOT contain an "alg". In HPKE, the algorithm used to encrypt the CEK is specified per-recipient in the recipient's header, making it unnecessary to include it in the JWE protected header.
```

Comment:
```suggestion
- The JWE protected header MUST NOT contain the "alg", when recipients use different algorithms to secure the content encryption key.

- The JWE protected header SHOULD contain the "alg", when all recipients use the same HPKE algorithm to secure the content encryption key.
```

Reply:
https://www.rfc-editor.org/rfc/rfc7516.html#section-7.2.1 says:
> Some Header Parameters, including the "alg" parameter, can be shared among all recipient computations.  Header Parameters in the JWE Protected Header and JWE Shared Unprotected Header values are shared among all recipients.

Orie is right that we cannot prohibit "alg" in the protected header.